src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs

```diff
@@ -45,6 +45,11 @@ private IdentityModelEventSource()
         /// </summary>
         public static string HiddenPIIString { get; } = "[PII of type '{0}' is hidden. For more details, see https://aka.ms/IdentityModel/PII.]";
 
+        /// <summary>
+        /// String that is used in place of any arguments to log messages if the 'LogCompleteSecurityArtifact' flag is set to false.
+        /// </summary>
+        public static string HiddenArtifactString { get; } = "[Security Artifact of type '{0}' is hidden. For more details, see https://aka.ms/IdentityModel/SecurityArtifactLogging.]";
```

Comment:
it is incredibly minor, but consider including a space after the link and after the period. I think that the period will be included in the link and wont work.

Reply:
Here I'm matching existing style (see `HiddenPIIString`) Seems like visual studio link generation and github email both don't include the period in the link and it looks more readable to my eye so leaving for now. Feel free to open a new issue and we can address outside this PR if you think it will cause problems.